packages/tools/pie-icons-react/turbo.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": [
+    "//"
+  ],
+  "pipeline": {
+    "clean": {
+      "cache": false,
+      "dependsOn": []
+    },
+    "build:icons": {
+      "cache": true,
+      "dependsOn": [
+        "clean"
+      ],
+      "outputs": [
+        "icons/**"
+      ]
+    },
+    "build": {
+      "cache": true,
+      "dependsOn": [
+        "^build",
```

Comment:
More as a noob turbo q – why do you need `^build` as a depends on of `build` here?

Reply:
This is a good question, I would have thought that this would be implied. I'll try removing it.

Reply:
I don't think turbo implies this relationship actually, so it seems to be necessary.